README.md

```diff
@@ -51,11 +51,11 @@ As said earlier, once this project leaves the examiners room, it will be open
 for all.
 
 P.S. If you do want to contribute to this project, you should open an issue asking to contribute
-and hopefully we can give you a feature to implement which is not in the above list.
+and hopefully, we can give you a feature to implement which is not in the above list.
 
 ## Post Scriptum
 
-Apologies, if this repository is not up to the standards of Open Source Software Community.
+Apologies, if this repository is not up to the standards of the Open Source Software Community.
```

Comment:
I don't think we need a "the" here.
WDYT ??

Reply:
Sir, I think the same, I would omit this!